tests/app/test_config.py

```diff
@@ -29,16 +30,22 @@ def reload_config(os_environ):
 
 def test_load_cloudfoundry_config_if_available(reload_config):
     os.environ['API_HOST_NAME'] = 'env'
+    os.environ['REDIS_URL'] = 'some uri'
```

Comment:
[thought] not really related to this PR, but I'm wondering if relying on `os.environ['REDIS_URL']` is why caching is broken in prod?

Reply:
I don't believe so, I've been able to verify (by logging it from the running remote just before attempting to write to the cache) that the REDIS_URL variable is correctly set with our broker uri (the connection string itself bears another look, though). 

It happens grabs the uri value from the VCAP_SERVICES json blob at the top of config.py. This is what the UK does in their CF deployment. I suppose something about our deployment could be unique and create a sequencing issue that creates the redis client before config, but from a cursory look at app/application.py and app/__init__.py, seems like config runs at the top. Could be interesting to set the REDIS_URL manually in the env vars, so it's available at build time and see if that fixes it